src/main/kotlin/com/sourcegraph/cody/config/BaseLoginDialog.kt

```diff
@@ -10,20 +9,54 @@ import com.intellij.openapi.project.Project
 import com.intellij.openapi.ui.DialogWrapper
 import com.intellij.openapi.ui.ValidationInfo
 import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.wm.IdeFocusManager
 import com.sourcegraph.cody.api.SourcegraphApiRequestExecutor
 import com.sourcegraph.cody.auth.SsoAuthMethod
 import java.awt.Component
+import java.awt.event.ActionEvent
+import javax.swing.Action
 import javax.swing.JComponent
 
 abstract class BaseLoginDialog(
-    project: Project?,
-    parent: Component?,
+    val project: Project?,
+    val parent: Component?,
     executorFactory: SourcegraphApiRequestExecutor.Factory,
     private val authMethod: SsoAuthMethod
 ) : DialogWrapper(project, parent, false, IdeModalityType.PROJECT) {
 
   protected val loginPanel = CodyLoginPanel(executorFactory)
 
+  private val authenticateAutomatically =
+      object : DialogWrapperAction("Authenticate automatically") {
+
+        override fun doAction(e: ActionEvent?) {
+          val info = loginPanel.getServerPathValidationInfo()
+          if (info != null) {
+            if (info.component != null && info.component!!.isVisible) {
+              IdeFocusManager.getInstance(null).requestFocus(info.component!!, true)
```

Comment:
Shouldn't it be `IdeFocusManager.getInstance(project)?`

Reply:
n/a